builders/build/substrate-api/sidecar.md

```diff
@@ -197,57 +197,57 @@ For more information on the new [EIP1559](https://eips.ethereum.org/EIPS/eip-155
 To obtain the EVM sender address, recipient address, and EVM hash of any EVM transaction type, check the `events` field under the current extrinsic object, and identify the event where the `method` field is set to:
 
 ```
-{event number}.method.pallet: "ethereum"
-{event number}.method.method: "Executed" 
+{event_number}.method.pallet: "ethereum"
+{event_number}.method.method: "Executed" 
```

Comment:
should this be in an array instead?

Reply:
In this two particular cases I'm using `{whatever}` because in the diagrams Henry had it like this. In the other case I'm using `[whatever]` because I think it is a cleaner way to access elements.